src/Character/Camera/DefaultCamera.lua

```diff
@@ -21,7 +21,51 @@ local Settings = NexusVRCharacterModel:GetInstance("State.Settings")
 local DefaultCamera = NexusObject:Extend()
 DefaultCamera:SetClassName("DefaultCamera")
 
+local ShownAccessories = {
```

Comment:
Two things:
1. If this is a global, should this be moved up and turned to `SHOWN_ACCESSORIES`.
2. I am guessing there was an explicit decision to use a whitelist instead of a blacklist. I plan to add a simple unit test to point out if the assumed values of `Enum.AccessoryType` change, but I am curious what the justification was to pick a whitelist.

Reply:
1. Will do.
2. I thought about it, probably would make more sense. I wrote most of that while I was sleep deprived 😅